styles.scss

```diff
@@ -646,7 +646,8 @@ button {
     flex-direction: column;
   }
 
-  .menu-card:hover {
+  .menu-card:hover,
+  .menu-card.clicked {
```

Comment:
We add the class of clicked to the menu card and then target the same hover styling